build/write-image.sh

```diff
@@ -21,10 +21,12 @@ sudo e2label ${OUTPUT_DEVICE}p4 blue
 mkdir -p /tmp/eos-mnt
 sudo mount ${OUTPUT_DEVICE}p1 /tmp/eos-mnt
 
+sudo sed -i 's/^/modprobe.blacklist=uas /g' /tmp/eos-mnt/cmdline.txt
```

Comment:
we should remove this if we add drives to our whitelist that we *know* work with UAS

Reply:
yeah we should remove this for sure.